<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>521</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="autoword_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="autoword_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="autoword_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="autoword_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="autoword_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ja_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㄱ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ja_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㄴ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ja_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㄷ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ja_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㄹ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ja_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㅁ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ja_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㅂ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mo_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㅛ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mo_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㅏ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mo_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㅗ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mo_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㅓ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mo_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㅕ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mo_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ㅑ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_space">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>space</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bspace">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>backspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_shift">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_question">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
